e2e/scenario_test.go

```diff
@@ -1971,3 +1971,113 @@ func Test_AzureLinux3OSGuard_PMC_Install(t *testing.T) {
 		},
 	})
 }
+
+func getDCGMPackageNames() []string {
+	return []string{
+		"datacenter-gpu-manager-4-core",
+		"datacenter-gpu-manager-4-proprietary",
+		"datacenter-gpu-manager-exporter",
+	}
+}
+
+func Test_Ubuntu2404NvidiaDCGMExporterRunning(t *testing.T) {
+	RunScenario(t, &Scenario{
+		Description: "Test that a Ubuntu 24.04 node has NVIDIA DCGM Exporter installed, running and scrapable",
+		Tags: Tags{
+			GPU: true,
+		},
+		Config: Config{
+			Cluster: ClusterKubenet,
+			VHD:     config.VHDUbuntu2404Gen2Containerd,
+			BootstrapConfigMutator: func(nbc *datamodel.NodeBootstrappingConfiguration) {
+				nbc.AgentPoolProfile.VMSize = "Standard_NC6s_v3"
+				nbc.ConfigGPUDriverIfNeeded = true
+				nbc.EnableGPUDevicePluginIfNeeded = false
+				nbc.EnableNvidia = true
+			},
+			VMConfigMutator: func(vmss *armcompute.VirtualMachineScaleSet) {
+				vmss.SKU.Name = to.Ptr("Standard_NC6s_v3")
+				if vmss.Tags == nil {
+					vmss.Tags = map[string]*string{}
+				}
+				vmss.Tags["EnableManagedGPUExperience"] = to.Ptr("true")
+			},
+			Validator: func(ctx context.Context, s *Scenario) {
+				for _, packageName := range getDCGMPackageNames() {
+					ValidateInstalledPackageVersion(ctx, s, packageName, components.GetExpectedPackageVersions(packageName, "ubuntu", "r2404")[0])
+				}
+
+				ValidateNvidiaDCGMExporterSystemDServiceRunning(ctx, s)
+				ValidateNvidiaDCGMExporterIsScrapable(ctx, s)
```

Comment:
What do you think about validating whether we can scrape a particular metric too?

Reply:
this could break if the new version of the dcgm exporter drops this metrics.  If we would be dependant on certain metrics for our own alerting logic, etc that could make sense